system/modules/core/classes/Backend.php

```diff
@@ -660,20 +660,21 @@ protected function getBackendModule($module)
 	 * @param integer $pid
 	 * @param string  $domain
 	 * @param boolean $blnIsSitemap
-	 * @param string  $strLanguage
 	 *
 	 * @return array
 	 */
-	public static function findSearchablePages($pid=0, $domain='', $blnIsSitemap=false, $strLanguage='')
+	public static function findSearchablePages($pid=0, $domain='', $blnIsSitemap=false)
 	{
-		$time = \Date::floorToMinute();
-		$objDatabase = \Database::getInstance();
-
-		// Get published pages
-		$objPages = $objDatabase->prepare("SELECT * FROM tl_page WHERE pid=? AND (start='' OR start<='$time') AND (stop='' OR stop>'" . ($time + 60) . "') AND published='1' ORDER BY sorting")
```

Comment:
You are telling me this based on something you think you remember, which however is nowhere documented?